src/main/webapp/resources/js/pages/analysis/components/sistr/CgMlst.jsx

```diff
@@ -25,19 +25,33 @@ export default function CgMlst({ sistrResults }) {
       {
         title: i18n("AnalysisSistr.percentMatching"),
         desc:
-          sistrResults.cgmlst_distance !== null
-            ? `${getPercentage(sistrResults.cgmlst_distance) + ""}%`
-            : ""
+            sistrResults.cgmlst_distance !== null
+                ? `${getPercentageFromDistance(sistrResults.cgmlst_distance) + ""}%`
+                : "-"
+      },
+      {
+        title: i18n("AnalysisSistr.allelesFoundGenome"),
+        desc: sistrResults.cgmlst_found_loci !== null
+          ? `${sistrResults.cgmlst_found_loci}/330` + "" : "-"
```

Comment:
I see the `330` magic number around in this file a few times.  Can that get changed to a defined variable at the top of the file or something instead of just `330`?

Reply:
Yes, I saw it too. I think it is good idea to define it as a constant variable which will make code modification easier as only a single place would need to be modified. This number refers to the maximum number of cgMLST alleles which should not change as SISTR uses this MLST scheme since the beginning. I do not see that this number would change unless a completely new cgMLST will be used in the future. Perhaps it is a good idea to add this field into SISTR results JSON file for automatic adjustment of this constant.